### PR TITLE
check in keyvalue example

### PR DESCRIPTION
**Pull request by blampe**
Renamed the `set` method to `setValue` to emphasize camel case (and `set` is a syntax error anyway).

I'm not a huge fan of checking in the generated code but it works out of the box.

@abhinav @junchaowu @breerly

### CODE REVIEW

File: examples/keyvalue/keyvalue/client.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+from service import KeyValue
+
+
+@gen.coroutine
+def run():
+    app_name = 'keyvalue-client'
+
+    app = TChannel(app_name)
+
+    client_class = client_for('keyvalue-server', KeyValue)
```
Comment (abhinav) [inline examples/keyvalue/keyvalue/client.py]: Make top-level as `KeyValueClient`

File: examples/keyvalue/keyvalue/client.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    client = client_class(app, 'localhost:8889')
```
Comment (abhinav) [inline examples/keyvalue/keyvalue/client.py]: Specifying the arg name might be nicer.

    hostport='localhost:8889'

**Comment by abhinav** [comment]
:+1: 

Minor comments

**Comment by junchaowu** [comment]
lgtm

**Pull request merged**